src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -109,6 +109,9 @@ def __init__(
             new_config["steps_offset"] = 1
             scheduler._internal_dict = FrozenDict(new_config)
 
+            self.text_encoder_name = text_encoder.config["architectures"].lower()
```

Comment:
Do we need this? Think for now just having: https://github.com/huggingface/diffusers/pull/2918/files#r1154437571 is good enough